docs/release-notes.md

```diff
@@ -10,6 +10,161 @@ toc_max_heading_level: 2
 
 Welcome to Semgrep release notes. This document gives an overview of the changes, additions, and selected fixes. Release notes published since April 2022 include Semgrep CLI, CI, and Semgrep App updates. Release notes published since April 2022 encompass all versions released during the month.
 
+## October 2022
+
+### Semgrep Supply Chain
+
+r2c now offers a new product: Semgrep Supply Chain, a software composition analysis solution. Semgrep Supply Chain is a high-signal dependency scanner that detects reachable vulnerabilities in open source, third-party libraries in your code. Reach out to sales@r2c.dev to learn more.
+
+### Semgrep App
+
+#### Additions
+
+- New demo project allows you to try out Semgrep App workflows. If your organization in Semgrep App does not have any projects assigned in the [Projects](https://semgrep.dev/orgs/-/projects) page, you can add the new demo project by clicking **Explore a demo project**.
+- You can now triage through PR comments, for more information see [Ignoring findings through comments](https://semgrep.dev/docs/semgrep-app/findings/#ignoring-findings-through-comments) documentation.
+- Semgrep Playground now displays its version number. To see the exact version that Semgrep Playground uses, click the <i class="fa-regular fa-ellipsis-vertical"></i> three-dot button, and then see the version number after the <i class="fa-solid fa-code-commit"></i> icon.
+
+    ![Semgrep Playground version number.](/img/semgrep-app-latest-version.png "Semgrep Playground version number.")
+
+#### Changes
+
+- Removing a rule automatically removes all findings that came from the removed rule. If you add a removed rule back, then you need to re-scan your code to get the findings from the previously removed rule again. For more information, see [Triaging findings](https://semgrep.dev/docs/semgrep-app/findings/#triaging-findings).
```

Comment:
@raghavjain3 Should we be noisier in the community Slack about this?

Reply:
@pabloest @adamkvitek we can message in the community slack! 

This is an extension of the existing behavior - right now you have to rescan to see the findings removed. With the change, they are automatically removed without rescan.

Reply:
Previously, when you removed a rule you had to rescan the code to remove findings produced by the rule. With this change, findings produced by the removed rule are now automatically removed without rescanning.